components/validation_messages/validation_messages.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     v-if="showMessages && !isFilteredValidationMessagesEmpty"
-    class="base-input__messages d-d-flex d-fd-column"
+    class="base-input__messages d-validation-message__container"
```

Comment:
Hmm couldn't find this class in dialtone, where is it stored?

Reply:
Oh, these are in `packages/dialtone/lib/build/less/components/forms.less`

Reply:
ah yeah didn't show up in my search because of `&__container`. Looks good thanks.